dtfs-central-api/src/v1/controllers/utilisation-report-service/get-utilisation-report-reconciliation-details-by-id.controller/helpers/get-utilisation-details.ts

```diff
@@ -0,0 +1,41 @@
+import { FeeRecordEntity, TfmFacility, FeeRecordUtilisation } from '@ukef/dtfs2-common';
+import { TfmFacilitiesRepo } from '../../../../../repositories/tfm-facilities-repo';
+import { NotFoundError } from '../../../../../errors';
+import { mapToFeeRecordUtilisation } from './map-to-fee-record-utilisation';
+
+/**
+ * Retrieves and constructs utilisation data for the given fee records
```

Comment:
more documentation here please.  For example returns all facility ids from fee record. then gets facilities etc

Reply:
This docstring was actually added in a previous approved PR that was only temporarily in the diff between the base getting merged into main and main getting merged into this. Is a quick change so will add more detail.